### ISSUE TITLE
2.2.3 added breaking change for default export

### ISSUE THREAD

**Issue by JingzheSun** [NorthwoodsSoftware/GoJS#170]
previous versions allow using default export e.g `import go from "gojs"`
but the newly added `"module": "release/go-module.js"` only works with named export `import * as go from "gojs"`.

It is expected that both import types are supported among the same minor version.
upgrading from 2.2.2 to 2.2.3 should not bring any other issue.

**Comment by simonsarris** [comment]
Thank you for pointing this out. We'll revert the package.json line.

**Comment by simonsarris** [comment]
If instead of reverting the package.json, if we modified go-module.js to use:

```js
export default go = (typeof global !== "undefined") ? global.go : self.go;
```

Instead of what it is right now:
```js
export const go = (typeof global !== "undefined") ? global.go : self.go;
```

Would that work for you? We want to make sure this is proper for module users.

**Comment by simonsarris** [comment]
We have just released 2.2.5, which should fix this. Please let me know if you continue to have issues.
